macros/models/_sector/nft/nft_transfers.sql

```diff
@@ -103,18 +91,11 @@ FROM(
     , et."from" AS executed_by
     {%- endif %}
     , t.evt_tx_hash AS tx_hash
-    , '{{blockchain}}' || cast(t.evt_tx_hash as varchar) || '-{{token_standard_1155}}-' || cast(t.contract_address as varchar) || '-' || cast(t.id as varchar) || '-' || cast(t."from" as varchar) || '-' || cast(t.to as varchar) || '-' || cast(t.value as varchar) || '-' || cast(t.evt_index as varchar) AS unique_transfer_id
     FROM (
         SELECT t.evt_block_time, t.evt_block_number, t.evt_tx_hash, t.contract_address, t."from", t.to, t.evt_index {% if denormalized == True %}, t.evt_tx_from {% endif %}
         , value, id
         FROM {{ erc1155_batch }} t
         CROSS JOIN unnest(zip(t."values", t.ids)) AS foo(value, id)
-        {% if is_incremental() %}
-        LEFT JOIN {{this}} anti_table
```

Comment:
hmm @0xRobin may have an idea why the `anti join` was originally used on spark? i forget off the top of the head

Reply:
it must've been since we used `append` in the incremental, rather than merge? but now that we set merge throughout, this can be removed?

Reply:
I think we initially did this on Spark because the merge was really slow when it contained this many rows.